src/routes/docs/products/functions/develop/+page.markdoc

```diff
@@ -683,6 +695,7 @@ These are provided alongside any custom headers sent to the function.
 |---------------------------|------------------------------------------------------------------------------------------------------------------------------------------------------|
 | `x-appwrite-trigger`        |  Describes how the function execution was invoked. Possible values are `http`, `schedule` or `event`.                                                      |
 | `x-appwrite-event`          |  If the function execution was triggered by an event, describes the triggering event.                                                                |
+| `x-appwrite-key`            |  API key used for server authentication. Your API key is a secret, do not use it in client applications.                                             |
```

Comment:
> Your API key is a secret, do not use it in client applications.

This warning sounds more applicable to a regular API key rather than a dynamic API key. How can you improve the warning? 

Also, how else can we provide more detailed info about this?